spec/support/test_helpers/candidate_helper.rb

```diff
@@ -22,6 +22,14 @@ def create_and_sign_in_candidate
     login_as(current_candidate)
   end
 
+  def application_form_sections
+    if FeatureFlag.active?(:reference_selection)
+      APPLICATION_FORM_SECTIONS - [:references_provided] + [:references_selected]
```

Comment:
:+1: Maybe update `APPLICATION_FORM_SECTIONS` to have the active list and toggle the logic here so that it's easier to remove the flag?

Reply:
Good idea 👍🏼